### DISCUSSION TITLE
v.1.0.0 Release Announcement

### DISCUSSION THREAD

**Discussion by 3xp11i** [prakhartiwari0/my-web-shortcuts#42] [Announcements]
# The journey has begun! My Web Shortcuts's first version is released :partying_face: :tada: 

This is the first version of My Web Shortcuts, it still contains many undiscovered bugs :bug:  and a lack of features, but I had to launch it just to see the response of people and if it's actually helping them or not. With their feedback, I can decide in which direction to go, and what features to add, as blindly adding anything I want is not a good thing to do. :sweat_smile: 

**Some unimportant but good-to-know info** :spiral_notepad: 
I submitted this version for review on 23 November at 1:30 A.M. IST, why I'm getting so specific? Because why not? It's my first ever real-world project! :wink:
At the time of writing this text (25 Nov 12:15 PM), no reply has come yet, so I don't know if it will be rejected or accepted. I was expecting this, and I think it will not come before 27 Nov at least, so in the meantime, I will create the website, usage guide & some YouTube tutorials, and share everything with the public. I want to see how the community is reacting.

I hope this tool adds value to your life, I would love to have your feedback :smile: 

Repeating what I said in the [Thoughts from the Creator](https://github.com/mywebshortcuts/mywebshortcuts/discussions/12) announcement, 

> *Whatever happens, what matters is that something new is created, something that adds value to people's lives, a new idea, new opportunities and new conversations. Collaboration and value addition should be our highest priority in Open Source.*

Thank you & Take care :heart: 

## What's Changed
* Update README.md by @jasonetorres in https://github.com/mywebshortcuts/mywebshortcuts/pull/8
* Create pull-request-template.md by @shubhsharma19 in https://github.com/mywebshortcuts/mywebshortcuts/pull/7
* Add VISION.md by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/17
* [docs] Improve README.md by @shubhsharma19 in https://github.com/mywebshortcuts/mywebshortcuts/pull/13
* Add Testing & Accessibility Issue templates by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/21
* Improve quality of the popup header logo image by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/24
* [docs] Improve grammar in Vision.md by @shubhsharma19 in https://github.com/mywebshortcuts/mywebshortcuts/pull/20
* Feat/options UI by @beingsie in https://github.com/mywebshortcuts/mywebshortcuts/pull/14
* Change Delete Button Text by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/26
* Deleted unneeded file common.css by @IrkenStudio in https://github.com/mywebshortcuts/mywebshortcuts/pull/27
* Build Basic Settings Page Layout by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/30
* Built & Improved Websites Lister in the Options Page by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/33
* Major Improvements to UI/UX of selector elements & setter.js code by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/34
* Improved selectors' UI/UX, inner functioning and selectors' functionality by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/35
* [ImgBot] Optimize images by @imgbot in https://github.com/mywebshortcuts/mywebshortcuts/pull/36
* Enhance Popup UI & inner functioning by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/38
* Options page by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/39
* Improved Options Page, new quick shortcut setting opener functionality and more by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/40
* Final changes & ready for production  by @prakhartiwari0 in https://github.com/mywebshortcuts/mywebshortcuts/pull/41

## New Contributors
* @jasonetorres made their first contribution in https://github.com/mywebshortcuts/mywebshortcuts/pull/8
* @shubhsharma19 made their first contribution in https://github.com/mywebshortcuts/mywebshortcuts/pull/7
* @prakhartiwari0 made their first contribution in https://github.com/mywebshortcuts/mywebshortcuts/pull/17
* @beingsie made their first contribution in https://github.com/mywebshortcuts/mywebshortcuts/pull/14
* @IrkenStudio made their first contribution in https://github.com/mywebshortcuts/mywebshortcuts/pull/27

**Full Changelog**: https://github.com/mywebshortcuts/mywebshortcuts/commits/v.1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mywebshortcuts/mywebshortcuts/releases/tag/v.1.0.0'>v.1.0.0</a>.</em>

**Comment by 3xp11i** [comment]
My Thanks to everyone who contributed to this project :heart: 

- Thanks @shubhsharma19 @jasonetorres for improving the documentation and starting the Open Source story of this project :rocket: 
- Thanks @beingsie for contributing so much to the UI/UX aspect of this project, your mockup UI was the basis of this extension's UI.
- Thanks @siddhant-khisty @ @RutamBhagat for testing the early versions of this extension and sharing your valuable feedback
- Thanks @xkrishguptaa for helping me solve a critical bug regarding input elements.
- Greetings to other contributors (@IrkenStudio and other folks who contributed indirectly)

**Reply by jasonetorres** [reply]
Proud to be a part of a cool project!

**Reply by 3xp11i** [reply]
Proud to have such amazing people as contributors! :fire:

**Reply by shubhxg** [reply]
You have done an amazing job on this great project @prakhartiwari0 !! Kudos to you for great work. 💜

**Reply by 3xp11i** [reply]
Thank you Shubh!

**Reply by siddhant-khisty** [reply]
Great work. It's just the beginning 😌 ✨️
